pkg/cluster/mutate_create_test.go

```diff
@@ -121,7 +196,7 @@ func TestClusterCreateMutate(t *testing.T) {
 func getCreateMutateAdmissionRequest(newMP []byte) *v1beta1.AdmissionRequest {
 	req := &v1beta1.AdmissionRequest{
 		Resource: metav1.GroupVersionResource{
-			Version:  "cluster.x-k8s.io/v1alpha3",
+			Version:  "cluster.x-k8s.io/v1alpha4",
```

Comment:
mm.m.. I don't think we are that advanced yet :sweat_smile:

Reply:
Fixed, thanks